monitoring/main.tf

```diff
@@ -0,0 +1,75 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    coder = {
+      source  = "coder/coder"
+      version = ">= 0.11"
+    }
+  }
+}
+
+variable "description" {
+  default     = "Monitoring of workspace resources"
+  description = "Monitoring of workspace resources"
+}
+
+variable "threshold" {
+  type        = number
+  description = "The threshold for the monitoring, used for all resources unless overridden by *_threshold."
+  default     = 90
+}
+
+variable "memory_threshold" {
+  type        = number
+  description = "The threshold for the memory monitoring."
+  default     = 90
+}
+
+variable "disk_threshold" {
+  type        = number
+  description = "The threshold for the disk monitoring."
+  default     = 90
+}
+
+variable "disks" {
+  type        = list(string)
+  description = "The disks to monitor."
+  default     = ["/"]
+}
+
+variable "enabled" {
```

Comment:
Why do we have a variable for this? 
Can use the terraform `count` to control if the module is added or not.

Reply:
I imagined we can use this enabled parameter to let users keep the block but just change the enabled variable if they want, for some specific cases, disable it quickly without loosing their configuration, does it make sense ? Otherwise I can delete it and indeed rely on the terraform count (I need to read about it.)

Reply:
That's the same purpose the `count` argument can serve. 
It can be set to 0 or 1 to enable or disable the whole module. 
If someone is not using the module they can use the count on the `data "coder_monitoring"` too. 
see: https://developer.hashicorp.com/terraform/language/meta-arguments/count